.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,19 @@
+name: Lighthouse CI tests
```

Comment:
Please keep in mind that this won't work on PRs from forks (which is majority of our PRs).

Reply:
Ah